.storybook/webpack.config.js

```diff
@@ -3,7 +3,27 @@ const path = require("path");
 module.exports = ({ config }) => {
   config.resolve.alias = {
     ...config.resolve.alias,
-    src: path.resolve(__dirname, "../packages/components/src"),
+    src: [
+      path.resolve(__dirname, "../packages/components/src"),
+      path.resolve(__dirname, "../packages/data-viz/src"),
+    ],
+  };
+
+  /**
+   * (masoudmanson): Due to a Storybook build issue in production mode, where importing
```

Comment:
Thanks for the note! Is there a Storybook Github issue we should link in the comment 😄 ? Thank you!

Reply:
Um, actually, there is no active Github issue for this problem. Maybe I should post an issue on the Storybook repo. 😯